docs/services/motion/algorithms.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "About the Motion Planning Algorithms"
+linkTitle: "Algorithms"
+weight: 20
+type: "docs"
+description: "Information about the motion planning algorithms Viam uses."
+---
+
+Viam implements two planning algorithms, both based in principle on [RRT](https://en.wikipedia.org/wiki/Rapidly-exploring_random_tree):
+
+## RRT*-Connect
+
+RRT*-Connect is an asymptotically optimal planner that samples the planning space randomly, connecting viable paths as it finds them.
+It continues sampling after it finds its first valid path, and if it finds future paths that are more efficient, it updates to report those instead.
+For Viam, efficiency/path quality is measured in terms of total kinematics state excursion.
+For an arm, this refers to joints; the total amount of joint change is minimized.
+For a gantry, this refers to the amount of linear movement.
+This algorithm can route around obstacles, but cannot satisfy topological constraints.
+
+## CBiRRT
+
+CBiRRT stands for Constrained, Bi-directional implementation of [RRT](https://en.wikipedia.org/wiki/Rapidly-exploring_random_tree).
+It creates paths which conform to specified constraints, and attempts to smooth them afterwards as needed.
+By default, it does not constrain the path of motion at all.
+This is to ensure that paths will be found when using defaults.
+CBiRRT returns the first valid path that it finds.
+The CBiRRT algorithm Viam uses is based on the algorithm described in this paper: <https://www.ri.cmu.edu/pub_files/2009/5/berenson_dmitry_2009_2.pdf>
+
+---
+
+By default, Viam's motion planning library uses a hybrid approach.

```

Comment:
Can I change this? If so how?

Reply:
@spieswl I have no idea

Reply:
You technically can, but we've specifically tried to avoid exposing this to the user so far. There are extras options which allow for more control over Motion Planning operation, but the `planning manager` that handles the hybrid approach was designed so that users would not need to get so deep into the technical details.

Reply:
So should I delete the "By default"?